distribution/solr-distro/src/main/resources/solrFiles/bin/solr.cmd

```diff
@@ -0,0 +1,2024 @@
+@REM
+@REM  Licensed to the Apache Software Foundation (ASF) under one or more
+@REM  contributor license agreements.  See the NOTICE file distributed with
+@REM  this work for additional information regarding copyright ownership.
+@REM  The ASF licenses this file to You under the Apache License, Version 2.0
+@REM  (the "License"); you may not use this file except in compliance with
+@REM  the License.  You may obtain a copy of the License at
+@REM
+@REM      http://www.apache.org/licenses/LICENSE-2.0
+@REM
+@REM  Unless required by applicable law or agreed to in writing, software
+@REM  distributed under the License is distributed on an "AS IS" BASIS,
+@REM  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+@REM  See the License for the specific language governing permissions and
+@REM  limitations under the License.
+
+@REM ************************************************************************************************************
+@REM This file differs from the Solr provided script based on the workaround posted on this forum
+@REM http://lucene.472066.n3.nabble.com/Logging-fails-when-starting-Solr-in-Windows-using-solr-cmd-td4396671.html
```

Comment:
@shaundmorris @rzwiefel 
The Solr devs are aware of this issue and admittedly don't spend time on windows issues. The change I made is outlined in this post so that we can carry it forward.